apps/laboratory/src/components/Ethers/EthersGetCallsStatusTest.tsx

```diff
@@ -13,15 +13,20 @@ import { W3mFrameProvider } from '@reown/appkit-wallet'
 import { type GetCallsStatusParams } from '../../types/EIP5792'
 import { EIP_5792_RPC_METHODS } from '../../utils/EIP5792Utils'
 
-export function EthersGetCallsStatusTest() {
+export function EthersGetCallsStatusTest(params: { callsHash: string }) {
```

Comment:
You can destructure from props so no need to use a variable on line 17.
```suggestion
export function EthersGetCallsStatusTest({ callsHash }: { callsHash: string }) {
```